catalog.html

```diff
@@ -113,10 +113,20 @@ <h2 class="product-sort-title">Сортировка:</h2>
                 <a class="product-sort-link" href="">По функционалу</a>
               </li>
               <li class="product-sort-item">
-                <a class="product-sort-link product-sort-more" href="">Больше</a>
+                <a class="product-sort-link product-sort-more sortActive" href="">
```

Comment:
_названия классов должны быть оформлены единообразно_
https://up.htmlacademy.ru/htmlcss/29/criteries#b15
если слова разделяются с помощью дефиса, то так должно быть везде
```suggestion
                <a class="product-sort-link product-sort-more sort-active" href="">
```

Reply:
Исправлено